### ISSUE TITLE
Reset Password call returns 400 for Not Found user

### ISSUE THREAD

**Issue by mkhatib** [lynndylanhurley/devise_token_auth#137]
Should return 404

```
Status Code:400 Bad Request
````

```
{"success":false,"errors":["Unable to find user with email 'example@gmail.com'."]}
```

**Comment by lynndylanhurley** [comment]
Thanks @mkhatib - this should be fixed as of `v0.1.32.beta3`.

**Comment by mkhatib** [comment]
Thanks for the fix! :-) Awesome work on the gem!
